scripts/install_stock.sh

```diff
@@ -37,7 +37,7 @@ wget --continue --quiet https://github.com/opencontainers/runc/releases/download
 mv runc.amd64 runc
 sudo install -D -m0755 runc /usr/local/sbin/runc
 
-wget --continue --quiet https://storage.googleapis.com/gvisor/releases/release/20210622/x86_64/runsc 
+wget --continue --quiet https://storage.googleapis.com/gvisor/releases/release/20221026/x86_64/runsc
```

Comment:
@CuriousGeorgiy why didn't you take a newer version of gVisor from 2023?

Reply:
Originally, I took the latest version — it didn't work. So I tried using older versions, but it didn't help.